src/store/modules/slide/mutation-types.js

```diff
@@ -55,3 +55,9 @@ export const SET_META_TIMEOUT = 'SET_META_TIMEOUT'
 export const SET_META_REPEATABLE = 'SET_META_REPEATABLE'
 export const SET_META_STARTDATE = 'SET_META_STARTDATE'
 export const SET_META_ENDDATE = 'SET_META_ENDDATE'
+
+// template which the slide should be displayed
+export const SET_TEMPLATE = 'SET_TEMPLATE'
+
+// timeout of the slide
+export const SET_TIMEOUT = 'SET_TIMEOUT'
```

Comment:
line 58-63 shouldn't be in this PR. We are removing these two mutation types

Reply:
Done